dll/ntdll/def/ntdll.spec

```diff
@@ -21,9 +21,9 @@
 @ stdcall CsrFreeCaptureBuffer(ptr)
 @ stdcall CsrGetProcessId()
 @ stdcall CsrIdentifyAlertableThread()
-@ stdcall CsrNewThread()
-@ stdcall CsrProbeForRead(ptr long long)
-@ stdcall CsrProbeForWrite(ptr long long)
+@ stdcall -version=0x502 CsrNewThread()
+@ stdcall -version=0x502 CsrProbeForRead(ptr long long)
+@ stdcall -version=0x502 CsrProbeForWrite(ptr long long)
```

Comment:
When using this syntax of `-version`, does this mean it's only just in NT 5.2 (e.g. in this example)? or does it mean its on all versions up to and including NT 5.2 ?

Reply:
Only this version. Other syntax is 0x500-0x502 or 0x600+

Reply:
OK. Then obviously there are a lot of fixes to do because e.g. these functions exist from NT 3.1 until Win2003 for example (see e.g. the link to Geoff Chappell)

Reply:
I don't care for anything below 2k3 atm.

Reply:
No problem; however there have been 2 or 3 functions down below that were labeled as existing from NT 4 to NT 5.2 and you specified the range for them :P

Reply:
I removed the 0x400-... stuff. For now I am not concerned about NT 4 compat. If we ever need it, we can add it, but then systematically.

Reply:
> don't care for anything below 2k3

Don't we care about XP at least? With regard to Workstation/Server settings, for example.
If not, then you can clean
`-version=0x501-0x502 LdrFindCreateProcessManifest`
too :-|

Reply:
Yes, but not now. We will eventually have a script to do this automatically.